.pre-commit-config.yaml

```
@@ -37,6 +37,7 @@ repos:
                 notebooks/api_examples/.*.ipynb|
                 notebooks/benchmarks/.*.ipynb|
                 notebooks/genomic_examples/.*.ipynb|
+                notebooks/tabular_examples/tree_based_models/League of Legends Win Prediction with XGBoost.ipynb|
```

This list is a regular expression, so I don't think the spaces will match correctly here. You could use a shorter pattern instead:

```suggestion
                notebooks/tabular_examples/tree_based_models/League.*.ipynb|
```

This is just a temporary measure whilst some notebooks are not yet clean.